src/utils/paths.ts

```diff
@@ -94,13 +101,12 @@ export class Win32Paths extends ProvidesResources implements Paths {
     throw new Error('lima not available for Windows');
   }
 
+  get oldIntegration(): string {
+    throw new Error('oldIntegration path not available for Windows');
```

Comment:
If this isn't an internal error, we need a better error message. Otherwise preface the message with "Internal error:" or something similar so everyone knows these should never be seen by users.